Dockerfile.dapper

```diff
@@ -100,16 +96,16 @@ RUN cd /usr/src && \
     git clone https://github.com/rancher/liblonghorn.git && \
     cd liblonghorn && \
     git checkout 09956ed9329d51f41a85d6e06113948c947976a8 && \
-    make deb && \
-    dpkg -i ./pkg/liblonghorn_*.deb
+    make && \
+    make install
 
 # Build TGT
 RUN cd /usr/src && \
     git clone https://github.com/rancher/tgt.git && \
     cd tgt && \
     git checkout e042fdd3616ca90619637b5826695a3de9b5dd8e && \
-    ./scripts/build-pkg.sh deb && \
-    dpkg -i ./pkg/tgt_*.deb
+    make && \
+    make install 
```

Comment:
There's an RPM spec file in https://github.com/rancher/tgt/blob/longhorn_oct2019/scripts/tgtd.spec.  We could build the RPM like how we build the debian package.

Reply:
Exactly, it seems the code is already ready and we can just adopt that.

For s390, I will merge this, and we can change it to rpm way later. (but the same, this is fine as well because the reason is the same as https://github.com/longhorn/longhorn-engine/pull/685#discussion_r851090253)

Reply:
cc @yangchiu @meldafrawi